### PR TITLE
g2o 2016-04-24 (new formula)

### PR DESCRIPTION
**Pull request by adrelino**
Added minimal g2o formula with tagged version 2016-04-24

### CODE REVIEW

**Comment by dpo** [comment]
Small audit:
```
==> audit problems
homebrew/science/g2o:
 * Description shouldn't start with an indefinite article (A)
```

File: g2o.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  depends_on "cmake" => :build
+  depends_on "eigen"
+  depends_on "suite-sparse" => :recommended
+  depends_on "qt" => :optional
+  depends_on "libqglviewer" => :optional
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make", "install"
```
Comment (dpo) [inline g2o.rb]: Are there unit tests that can be run at install time?

File: g2o.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    end
+  end
+
+  test do
+    system "#{bin}/g2o", "-v"
```
Comment (dpo) [inline g2o.rb]: Isn't there a better test than this? Ideally, a test should be a sample program that links with the library.

**Comment by adrelino** [comment]
Thanks for closing this! I didn't have enough experience to properly write the install and test targets.

**Pull request closed without merging**